src/ExceptionVisualizer/ExceptionVisualizer.csproj

```diff
@@ -20,22 +20,18 @@
 	<ItemGroup>
 		<Page Remove="ExceptionUserControl.xaml" />
 		<EmbeddedResource Include="ExceptionUserControl.xaml" LogicalName="$(RootNamespace).ExceptionUserControl.xaml" />
-
-		<Content Include="..\ExceptionVisualizerSource\bin\$(Configuration)\netstandard2.0\ExceptionVisualizerSource.dll" Link="netstandard2.0\ExceptionVisualizerSource.dll">
-			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-		</Content>
-		<Content Include="..\ExceptionVisualizerSource\bin\$(Configuration)\netstandard2.0\Ben.Demystifier.dll" Link="netstandard2.0\Ben.Demystifier.dll">
-			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-		</Content>
-
-		<None Remove="icon.png" />
-		<Content Include="icon.png">
-			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
-		</Content>
+		<Content Include="icon.png" Visible="false" />
```

Comment:
I cleaned up the icon part a bit. It is just less code. It still gets included in VSIX